GameOfLife/GameOfLife/Iteration.cs

```diff
@@ -0,0 +1,156 @@
+﻿namespace GameOfLife
+{
+    public class Iteration
+    {
+        private List<Tuple<int, int>> cellsToDie = new List<Tuple<int, int>>();
+        private List<Tuple<int, int>> cellsToBeBorn = new List<Tuple<int, int>>();
```

Comment:
You can create named valued with Tuple like

```
private List<(int x, int y)> _cellsToDie = new List<(int x, int y)>();
````

This way when trying to get value from them you can actually call them by name instead of cellsToDie.Item1 you can use cellsToDie.x

Reply:
This way the code looks much better! Changed the way of creation of the lists of Tuple.

Reply:
Can you add me as as a collaborator again? I think I missed your invite :P
That's why I cant resolve comments and looks like my approve doesn't count :D

Reply:
Okay, will do, just a second)

Reply:
Added.